SeleniumTest/src/test/java/com/XpathCss/XpathProject

```diff
@@ -0,0 +1,7 @@
+1.driver.findElement(By.xpath("//input[@id='userName']");
+driver.findElements(By.xpath("//button"));
+driver.findElement(By.xpath("//div[text()='Vega']"));
```

Comment:
Ensure that you search within the context of the table

//div[@class='ReactTable -striped -highlight']//div[text()='Vega']